### PR TITLE
Smart quotes better version, handles `len("“")` correctly instead of brute force full replace

### PR DESCRIPTION
**Pull request by ldemailly**

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -161,6 +161,33 @@ func DurationString(d time.Duration) string {
 	return strconv.Itoa(days) + "d" + rounded.String()
 }
 
+const smartQuotes = "“”"
```
Comment (ccoVeille) [inline bot.go:164]: I would have used backtick, plus a comment

```suggestion
const smartQuotes = `“”` // not regular double quotes
```

Comment (ccoVeille) [inline bot.go:164]: So you are no longer replacing "smart single quote" ?

**Review by ccoVeille — COMMENTED** [review]
Wow, tricky indeed.

I would have thought replacing them all was enough. Good catch

File: bot.go (hunk 1 above)
Comment (ldemailly) [reply bot.go:164]: yes they are hard to differentiate on screen, good point (though hopefully the variable name
gives a hint)

**Pull request merged**